.github/workflows/release.yml

```diff
@@ -42,3 +42,11 @@ jobs:
           PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
           SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
           SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+
+      - name: Build fat jar
+        run: sbt clean assembly
+      - name: Save fat jar artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: ergo-appkit.jar
+          path: target/**/ergo-appkit-*.jar
```

Comment:
What this action do?

Reply:
Takes every ergo-appkit-*.jar file (the fat jar produced by sbt assembly) and saves it as an artifact on the release action, so everyone can download and use it without building on their own.
Convenient for Python users not wanting to set up sbt.